doc/classes/Basis.xml

```diff
@@ -72,10 +72,18 @@
 			<description>
 				Constructs a pure rotation Basis matrix from Euler angles in the specified Euler rotation order. By default, use YXZ order (most common). See the [enum EulerOrder] enum for possible values.
 				[codeblock]
```

Comment:
```suggestion
				[codeblocks]
```
Same everywhere

Reply:
Oops, my bad. For future reference, is running `doc/tools/doc_status.py` locally a good way to check everything is formatted correctly?

Reply:
Haven't tried, might give cryptic results like the CI (which doesn't tell you what's wrong in this case for example)

Reply:
Not `doc_status.py`, but `make_rst.py` itself reports the errors. Provided you have GNU make and python locally, you can run `make rst` in the `doc` folder.

Reply:
Noted, thanks.